### DISCUSSION TITLE
list of pdfs as input

### DISCUSSION THREAD

**Discussion by Abdulk084** [nlmatics/llmsherpa#21] [Q&A]
`pdf_url = "https://arxiv.org/pdf/1910.13461.pdf"`

does it support list or only a single pdf?

**Comment by ansukla** [comment]
Hi @Abdulk084,

Currently it only supports a single pdf at a time. You can write a loop like this to parse multiple files.

folder = #enter folder path here
for file_name in os.listdir(folder):
    f = os.path.join(folder, file_name)
    # checking if it is a file
    if os.path.isfile(f):
           #call read file here

**Comment by ansukla** [comment]
Closing this Issue.
